### PR TITLE
docs: adds actionmenu and dropdown pages

### PR DESCRIPTION
**Pull request by EdwardIrby**

### CODE REVIEW

File: docs/src/components/code-block/package-map.ts [hunk 1]
```diff
@@ -65,6 +69,7 @@ export const PACKAGE_MAP: PackageMap = {
   '@pluralsight/ps-design-system-banner': { Banner },
   '@pluralsight/ps-design-system-breadcrumb': { Breadcrumb },
   '@pluralsight/ps-design-system-button': { Button },
+  '@pluralsight/ps-design-system-core': { core },
```
Comment (jaketrent) [inline docs/src/components/code-block/package-map.ts:72]: I think we want to encourage people to use the named imports so they get tree shaking.   As in this circ prog PR: https://github.com/pluralsight/design-system/blob/4dfceab1f5af46fb3bbb9578d1f114661f415795/docs/src/components/code-block/package-map.ts#L73

File: docs/src/pages/components/actionmenu.mdx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+        </ActionMenu.Icon>
+        Three item
+      </ActionMenu.Item>
+    </ActionMenu>
+}
+export default Example
+```
+
+### Ellipsis
+Use <code>ActionMenu.Ellipsis</code> to add ellipsis to action menu items.
```
Comment (jaketrent) [inline docs/src/pages/components/actionmenu.mdx:52]: Use backticks for inline code.

File: docs/src/pages/components/actionmenu.mdx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+      <ActionMenu.Divider />
+      <ActionMenu.Item>
+        Four item
+      </ActionMenu.Item>
+    </ActionMenu>
+}
+export default Example
+```
+
+### Nesting
```
Comment (jaketrent) [inline docs/src/pages/components/actionmenu.mdx:104]: Style-wise, I'd put a line break after this to be consistent with other mdx

**Review by jaketrent — APPROVED** [review]

File: docs/src/pages/components/actionmenu.mdx (hunk 3 above)
Comment (EdwardIrby) [reply docs/src/pages/components/actionmenu.mdx:104]: Cool

File: docs/src/pages/components/actionmenu.mdx (hunk 2 above)
Comment (EdwardIrby) [reply docs/src/pages/components/actionmenu.mdx:52]: Awe thanks for that.

File: docs/src/components/code-block/package-map.ts (hunk 1 above)
Comment (EdwardIrby) [reply docs/src/components/code-block/package-map.ts:72]: I'll see if I can make that work. It worked great for Icon but got an error when using it on core.

**Pull request merged**